.env

```
@@ -0,0 +1 @@
+VITE_GSA_URL="https://script.google.com/macros/s/AKfycbxee9OQhJV0udREFodBgVHnPMJR8Q_UJpq4j2ATUiBOND7hLA3Z6_ytr1YIWCrxn5z7/exec"
```

```suggestion
VITE_GSA_URL="https://script.google.com/macros/s/AKfycbyOkg88AKp3R-F68H9R7JWlHvyZaNUfKA82n4Dy117QmXWDItyyPdtM_6FU8paZLoCN/exec"
```

New URL shared by Mohit.